pom.xml

```diff
@@ -1090,6 +1111,58 @@
                 <version>${spring.version}</version>
             </dependency>
 
+            <dependency>
+                <artifactId>spring-expression</artifactId>
+                <groupId>org.springframework</groupId>
+                <version>${spring.version}</version>
+            </dependency>
+
+            <dependency>
+                <artifactId>spring-test</artifactId>
+                <groupId>org.springframework</groupId>
+                <version>${spring.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-test</artifactId>
+                <version>${spring-boot.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.apache.solr</groupId>
+                <artifactId>solr-cell</artifactId>
+                <version>${solr.client.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.lucene</groupId>
+                <artifactId>lucene-analyzers-icu</artifactId>
+                <version>${solr.client.version}</version>
```

Comment:
again test scope. ALso PLEASE bring over the important comment from server-webapp's POM